### PR TITLE
Seth is tired

### PR DESCRIPTION
**Pull request by Telesphoreo**

### CODE REVIEW

**Comment by ncurran02** [comment]
Umm why? It's fine with the way it is.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_rainbowtag.java [hunk 1]
```diff
@@ -7,6 +7,7 @@
 import org.bukkit.command.Command;
 import org.bukkit.command.CommandSender;
 import org.bukkit.entity.Player;
+import static me.totalfreedom.totalfreedommod.command.Command_tag.FORBIDDEN_WORDS;
```
Comment (OxLemonxO) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_rainbowtag.java]: remove unused import

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_tagnyan.java [hunk 2]
```diff
@@ -8,6 +8,7 @@
 import org.bukkit.command.Command;
 import org.bukkit.command.CommandSender;
 import org.bukkit.entity.Player;
+import static me.totalfreedom.totalfreedommod.command.Command_tag.FORBIDDEN_WORDS;
```
Comment (OxLemonxO) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_tagnyan.java]: remove unused import

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_rainbowtag.java [hunk 3]
```diff
@@ -29,7 +30,7 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
             return true;
         }
 
-        for (String word : Command_tag.FORBIDDEN_WORDS)
+        for (String word : FORBIDDEN_WORDS)
```
Comment (OxLemonxO) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_rainbowtag.java]: it's still command_tag

**Pull request closed without merging**